cdap-data-fabric/src/test/java/co/cask/cdap/data2/transaction/TransactionContextTest.java

```diff
@@ -630,13 +630,7 @@ public String getTransactionAwareName() {
 
     @Override
     public boolean canCommit(Transaction tx, Collection<byte[]> changeIds) throws TransactionNotInProgressException {
-      if (failCanCommitOnce) {
-        failCanCommitOnce = false;
-        return false;
-      } else {
-        //noinspection deprecation
-        return super.canCommit(tx, changeIds);
-      }
+      throw new RuntimeException("This should never be called)");
```

Comment:
nit - why the trailing parentheses? Same for the change below on line 642.
Other changes in this commit LGTM